src/query/api/v1/handler/placement/delete_all.go

```diff
@@ -23,18 +23,26 @@ package placement
 import (
 	"encoding/json"
 	"net/http"
+	"time"
 
-	"github.com/m3db/m3/src/cmd/services/m3query/config"
 	"github.com/m3db/m3/src/query/api/v1/handler"
 	"github.com/m3db/m3/src/query/util/logging"
-	clusterclient "github.com/m3db/m3cluster/client"
 
 	"go.uber.org/zap"
 )
 
 const (
-	// DeleteAllURL is the url for the handler to delete all placements (with the DELETE method).
-	DeleteAllURL = handler.RoutePrefixV1 + "/placement"
+	// DeprecatedM3DBDeleteAllURL is the old url for the handler to delete all placements, maintained
+	// for backwards compatibility.
+	DeprecatedM3DBDeleteAllURL = handler.RoutePrefixV1 + "/placement"
+
+	// M3DBDeleteAllURL is the url for the handler to delete all placements (with the DELETE method)
+	// for the M3DB service.
+	M3DBDeleteAllURL = handler.RoutePrefixV1 + "/services/m3db/placement"
```

Comment:
This could be reused from somewhere yeah?  `"/services/m3db/placement"` is rewritten a fair bit.